Documentation/CRDs/Cluster/external-cluster/consumer-import.md

```diff
@@ -23,7 +23,7 @@ helm install --create-namespace --namespace $clusterNamespace rook-ceph-cluster
 
 If not installing with Helm, here are the steps to install with manifests.
 
-1. Deploy Rook, create [common.yaml](https://github.com/rook/rook/blob/master/deploy/examples/common.yaml), [crds.yaml](https://github.com/rook/rook/blob/master/deploy/examples/crds.yaml) and [operator.yaml](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml) manifests.
+1. Deploy Rook, create [common.yaml](https://github.com/rook/rook/blob/master/deploy/examples/common.yaml), [crds.yaml](https://github.com/rook/rook/blob/master/deploy/examples/crds.yaml), [csi-operator.yaml](https://github.com/rook/rook/blob/master/deploy/examples/csi-operator.yaml) and [operator.yaml](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml) manifests.
```

Comment:
@subhamkrai, for Helm installs, do we need to make any changes? or is it auto-installed by rook ceph  chart?

Reply:
No csi-operator chart are installed as part of rook chart dependencies only